common/src/main/scala/DirectorsResult.scala

```diff
@@ -43,7 +43,7 @@ case class DirectorsResult(
   import DirectorsResult._
 
   val (period, periodQuantity) =
-    Period.Week -> Interval(from, to).numberOfWeeks().getOrElse(sys.error(s"Could not work  number of weeks between $from and $to"))
+    Period.Week -> (Interval(from, to).numberOfWeeks().getOrElse(sys.error(s"Could not work  number of weeks between $from and $to")) -1)
```

Comment:
when testing for the tax year 2007 the limits `5th April 2007 - 6th April 2008` gave 53 weeks instead of 52. The `- 1` is a hack to get it working but  `.numberOfWeeks` should probably be revisited

Reply:
sorted it out now